Program_#1_McNerthney.cpp

```diff
@@ -0,0 +1,125 @@
+#include <iostream>
+using namespace std;
+// Nolan McNerthney, CS 162, Project #1 
+
+
+int main()
+{
+	//To see if the program should terminate	
+	char run_program{'Y'};  
+
+
+	//Welcome the user and explain how the program works
+        cout << "Welcome to the planning app!" << endl
+	<<	"This app will determine if your daily workload is right for you." << endl
+	<<	"This is done by asking how many hours you want to work, how many projects you want to work on, and how many hours a day you want to work on each project." << endl
+	<<	"Then the app will calculate if the total hours working on projects is in an appropriate range. Let's begin!" << endl; 
+
+	//Run the program
+	while (toupper(run_program) == 'Y'){                   
+
+		//Define variables
+		char project_number_comf{'N'}; //Confirm the number of projects
+		char get_total_hours{1};       //To loop getting total hours till an appropriate value is inputted
+		char get_project_number{1};    //To loop getting 	
+		int project_number{0};         //Keeps track of the amount of projects  
+        	int count{1};                  //A counter that will go up as users enter info on each project
+       		float project_hours{0};          //The amount of hours for one project
+		float total_hours{0};            //The amount of hours for all project                	
+		float  hours_day{0};              //The amount of hours the user wants to work in a day
+
+
+		//Get total hours per day
+		while(get_total_hours == 1){
```

Comment:
Using a `bool` makes for cleaner syntax
```suggestion
		while(get_total_hours){
```